xport.py

```diff
@@ -163,7 +163,9 @@ def __init__(self, fp):
             self.created = created
             self.modified = modified
 
-            namestr_size = self._read_member_header()[-1]
+            dsname, _, dslabel, _, _, _, _, namestr_size = self._read_member_header()
+            self.dsname = dsname
```

Comment:
I'm not sure that `dsname` and `dslabel` are the best attribute names. Regardless of whether you can come up with something better, do you mind adding a change to the README to explain these?

Another option is making them properties to allow a docstring for them separate from the class.